.jest-test-results.json

```diff
@@ -0,0 +1 @@
":0,"numFailedTests":0,"numPassedTestSuites":4,"numPassedTests":6,"numPendingTestSuites":0,"numPendingTests":0,"numRuntimeErrorTestSuites":0,"numTotalTestSuites":4,"numTotalTests":6,"openHandles":[],"snapshot":{"added":0,"didUpdate":false,"failure":false,"filesAdded":0,"filesRemoved":0,"filesUnmatched":0,"filesUpdated":0,"matched":2,"total":2,"unchecked":0,"uncheckedKeysByFile":[],"unmatched":0,"updated":0},"startTime":1532718890169,"success":true,"testResults":[{"assertionResults":[{"ancestorTitles":["lattice-ui-kit named exports"],"failureMessages":[],"fullName":"lattice-ui-kit named exports should export the correct version","location":null,"status":"passed","title":"should export the correct version"}],"endTime":1532718892428,"message":"","name":"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/index_named_exports.test.js","startTime":1532718891828,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["lattice-ui-kit default export"],"failureMessages":[],"fullName":"lattice-ui-kit default export should export the correct version","location":null,"status":"passed","title":"should export the correct version"}],"endTime":1532718892476,"message":"","name":"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/index_default_export.test.js","startTime":1532718891864,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["Select"],"failureMessages":[],"fullName":"Select render matches snapshot","location":null,"status":"passed","title":"render matches snapshot"},{"ancestorTitles":["Select"],"failureMessages":[],"fullName":"Select attrs classNamePrefix is set to \"lattice-select\"","location":null,"status":"passed","title":"attrs classNamePrefix is set to \"lattice-select\""}],"endTime":1532718892487,"message":"","name":"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/select/src/components/__tests__/Select.test.js","startTime":1532718891794,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["Creatable"],"failureMessages":[],"fullName":"Creatable render matches snapshot","location":null,"status":"passed","title":"render matches snapshot"},{"ancestorTitles":["Creatable"],"failureMessages":[],"fullName":"Creatable attrs classNamePrefix is set to \"lattice-select\"","location":null,"status":"passed","title":"attrs classNamePrefix is set to \"lattice-select\""}],"endTime":1532718892491,"message":"","name":"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/select/src/components/__tests__/Creatable.test.js","startTime":1532718891836,"status":"passed","summary":""}],"wasInterrupted":false,"coverageMap":{"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/index.js":{"path":"/Users/solomon/openlattice-frontend/lattice-ui-kit/src/index.js","statementMap":{"0":{"start":{"line":7,"column":24},"end":{"line":7,"column":35}}},"fnMap":{},"branchMap":{},"s":{"0":2},"f":{},"b":{},"_coverageSchema":"332fd63041d2c1bcb487cc26dd0d5f7d97098a6c","hash":"22a5e96acc0adab2f18d34461ee63f5f56c905f2"}}}
```

Comment:
What's this?

Reply:
This was generated by the storybook jest-addon. I will remove it since the addon isn't working properly anyway.